File: 백준/Bronze/10987. 모음의 개수/모음의 개수.cc
#define _CRT_SECURE_NO_WARNINGS
#pragma warning(disable: 4996)
#include <iostream>
#include <string>
#define endl '\n'
using namespace std;

//백준 10987번 코드
int main() {
	ios::sync_with_stdio(false);
	cin.tie(NULL);	cout.tie(NULL);

	string st;
	cin >> st;

	int count = 0;
	for (int i = 0; i < st.length(); i++) {
		if (st[i] == 'a' || st[i] == 'e' || st[i] == 'i' || st[i] == 'o' || st[i] == 'u') {
			count++;
		}
	}
	cout << count << endl;

}

File: 백준/Bronze/10987. 모음의 개수/모음의 개수.cs
 string str = Console.ReadLine();

int count = 0;

for (int i = 0; i < str.Length; i++)
{
    char t = str[i];
    
    if (t == 'a' || t == 'e' || t == 'i' || t == 'o' || t == 'u')
        ++count;
}

Console.WriteLine(count);